CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Stream Chat Swift SDK
+
+We're glad you want to contribute to the Stream team 🎉
+
+---
+
+_So you.._
```

Comment:
> So you..
Add the 3rd dot: it should be: `So you...`

And the last thing, can you update `CHANGELOG.md`:
- replace `##### Breaking changes` with `## Breaking changes`,
- then replace all `##` to `###`.